src/libs/PolicyUtils.js

```diff
@@ -10,7 +10,7 @@ import ONYXKEYS from '../ONYXKEYS';
  * @returns {Array}
  */
 function getActivePolicies(policies) {
-    return _.filter(policies, (policy) => policy && policy.isPolicyExpenseChatEnabled && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
+    return _.filter(policies, (policy) => policy && (policy.isPolicyExpenseChatEnabled || policy.areChatRoomsEnabled) && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
```

Comment:
Would you mind updating the comment in this doc to explain what an "active policy" is? OldDot and NewDot seems to have different interpretations of this yet use the same language.

Reply:
Good point it seems we have completely different definitions here